README.md

```diff
@@ -242,6 +240,48 @@ Sometimes it's useful to get all changes for all repos e.g. to support finding t
 make pull
 ```
 
+### How to: set up Dnsmasq manually
```

Comment:
I wonder if we should add a line to the top of this section with something like "This is only necessary if `govuk-docker setup` doesn't work" in case someone spots this without knowing about `govuk-docker setup`.

But I like that we've still got this here because there are situations where the `setup` command won't work.

Reply:
Yep, I had to follow the "Then create or append to `/usr/local/etc/dnsmasq.d/development.conf`:" step personally.

Reply:
Now added a line 👍